bucket/carotdav.json

```
@@ -0,0 +1,30 @@
+{
+    "homepage": "http://rei.to/carotdav_en.html",
+    "description": "A simple WebDAV / FTP / SFTP / Online Storages client for Windows OS",
+    "license": {
+        "url": "http://rei.to/carotdav_en.html#license"
+    },
+    "version": "1.15.5",
+    "url": "http://rei.to/CarotDAV1.15.5.portable.zip",
+    "hash": "21b0b68ec45223c747d67b0e21ed87e8c195d52e5fc651b30b631a2f328bcd01",
+    "extract_dir": "CarotDAV",
+    "bin": [
+        "CarotDAV.exe",
+        "CarotDAVC.exe"
+    ],
+    "persist": "Setting.xml",
+    "pre_install": "if(!(Test-Path \"$persist_dir\\Setting.xml\")) {Set-Content -Path (\"$dir\\Setting.xml\") -Value ('<?xml version=\"1.0\" encoding=\"utf-8\"?><WebDAVClientSetting xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns:xsd=\"http://www.w3.org/2001/XMLSchema\"></WebDAVClientSetting>')}",
```

Why is value array with only 1 string?

---

It's the base config for CarotDAV and line break is unnecessary in XML.

---

But then you do not need array. It could be just string
```suggestion
    "pre_install": "if(!(Test-Path \"$persist_dir\\Setting.xml\")) {Set-Content -Path \"$dir\\Setting.xml\" -Value '<?xml version=\"1.0\" encoding=\"utf-8\"?><WebDAVClientSetting xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns:xsd=\"http://www.w3.org/2001/XMLSchema\"></WebDAVClientSetting>'}",
```

---

😂